cudapoa/src/application_parameters.cpp

```diff
@@ -106,6 +110,17 @@ ApplicationParameters::ApplicationParameters(int argc, char* argv[])
         throw std::runtime_error("band-width must be positive");
     }
 
+    if (!banded && adaptive)
+    {
+        throw std::runtime_error("adaptive alignment cannot run with full alignment");
+    }
+
+    if (adaptive && default_bandwidth)
+    {
+        // this is used to determine total score matrix size in adaptive-banded alignment
+        band_width = 2048;
```

Comment:
We can remove this code if 256 is alright for bandwidth.

Reply:
yes, 256 is good for the current one for 2 reasons: 1- average band-widths in most cases is around 128, 2- score matrix allocation is based on `band_width*max_number_of_nodes_per_window`, in most cases graph length is smaller than `max_number_of_nodes_per_window` and even if average is adaptive band-width is more than 256, it still can run without problem.

Reply:
although, if we change the heuristic rules in `set_adaptive_band_parameters()` we may need to adjust this default band-width again.